app-admin/yadm/yadm-3.2.1.ebuild

```
@@ -0,0 +1,90 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit bash-completion-r1 python-any-r1 python-utils-r1
+
+DESCRIPTION="Git based tool for managing dotfiles"
+HOMEPAGE="https://github.com/TheLocehiliosan/yadm"
+SRC_URI="https://github.com/TheLocehiliosan/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-shells/bash
+	app-crypt/gnupg
+	dev-vcs/git
+"
+
+BDEPEND="
+	test? (
+		${RDEPEND}
+		${PYTHON_DEPS}
+		$(python_gen_any_dep '
+			dev-python/pytest[${PYTHON_USEDEP}]
+		')
+		dev-tcltk/expect
+	)
+"
+
+DOCS=(CHANGES README.md ${PN}.md)
+
+python_check_deps() {
+	has_version -b "dev-python/pytest[${PYTHON_USEDEP}]"
+}
+
+pkg_setup() {
+	use test && python-any-r1_pkg_setup
+}
+
+src_compile() {
+	emake "${PN}.md"
+}
+
+src_test() {
+	# prevent system config having influence on tests
+	local -x GIT_CONFIG_NOSYSTEM=1
+
+	# prevent git branch warning, it confuses tests
+	local -x GIT_CONFIG_GLOBAL="${HOME}/.gitconfig"
+	git config --global init.defaultBranch master || die "setting default branch name failed"
+
+	# un-hardcode path to tmp, otherwise encryption tests fail
+	sed -i "s@^cache_dir = /tmp@cache_dir = ${TMPDIR}@" pytest.ini || die
```

We can improve a little the `sed`:
```suggestion
	sed -e "/^cache_dir/s@/tmp@${T}@" -i pytest.ini || die
```

About the tmpdir change. Based on [pms](https://projects.gentoo.org/pms/8/pms.html#x1-10900011.1), it is preferable to use `${T}` over `${TEMPDIR}`